CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
I'm not sure how to make this into a "suggestion", but I assume this big block of newlines at the top was unintentional and should be removed. :smile: